From bunk@stusta.de Thu Dec 29 11:07:28 2005
Date: Thu, 29 Dec 2005 20:07:25 +0100
From: Adrian Bunk <bunk@stusta.de>
To: gregkh@suse.de
Subject: PCI: schedule PCI_LEGACY_PROC for removal
Message-ID: <20051229190725.GO3811@stusta.de>
Content-Disposition: inline

PCI_LEGACY_PROC is deprecated since 2.5.53 in favor of lspci(8).


Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/feature-removal-schedule.txt |    7 +++++++
 1 file changed, 7 insertions(+)

--- gregkh-2.6.orig/Documentation/feature-removal-schedule.txt
+++ gregkh-2.6/Documentation/feature-removal-schedule.txt
@@ -171,3 +171,10 @@ Why:	The ISA interface is faster and sho
 	probing is also known to cause trouble in at least one case (see
 	bug #5889.)
 Who:	Jean Delvare <khali@linux-fr.org>
+
+---------------------------
+
+What:	Legacy /proc/pci interface (PCI_LEGACY_PROC)
+When:	March 2006
+Why:	deprecated since 2.5.53 in favor of lspci(8)
+Who:	Adrian Bunk <bunk@stusta.de>
